### PR TITLE
Bulk Validation: missing alt tags, sensitive language, dupe titles an…

### PR DESCRIPTION
**Pull request by jmirabal80**
…d H1

### CODE REVIEW

**Comment by jmirabal80** [comment]
Global effort to fix build validation errors

@Rick-Anderson 

The Content & Learning team is fixing build validation errors on docs.microsoft.com for the rest of H2. This will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. Please review within five business days and merge or comment in the PR with any changes you’d like to see. Thank you!

File: aspnet/identity/overview/getting-started/introduction-to-aspnet-identity.md [hunk 1]
```diff
@@ -93,7 +93,7 @@ ASP.NET Identity is implemented using the following procedure. The purpose of th
 
 1. Create an ASP.NET MVC application with Individual Accounts. You can use ASP.NET Identity in ASP.NET MVC, Web Forms, Web API, SignalR etc. In this article we will start with an ASP.NET MVC application.  
   
-    ![](introduction-to-aspnet-identity/_static/image1.png)
+    ![Image of new A S P dot Net project window](introduction-to-aspnet-identity/_static/image1.png)
```
Comment (Rick-Anderson) [inline aspnet/identity/overview/getting-started/introduction-to-aspnet-identity.md:96]: It's ASP, not `A S P`

File: aspnet/identity/overview/migrations/migrating-an-existing-website-from-sql-membership-to-aspnet-identity.md [hunk 2]
```diff
@@ -24,33 +24,33 @@ For this tutorial, we will take a web application template (Web Forms) created u
 
 1. We need to start with an existing application that uses SQL membership and has user and role data. For the purpose of this article, let's create a web application in Visual Studio 2010.
 
-    ![](migrating-an-existing-website-from-sql-membership-to-aspnet-identity/_static/image1.jpg)
+    ![Image of create new project window](migrating-an-existing-website-from-sql-membership-to-aspnet-identity/_static/image1.jpg)
 2. Using the ASP.NET Configuration tool, create 2 users: **oldAdminUser** and **oldUser.**
 
-    ![](migrating-an-existing-website-from-sql-membership-to-aspnet-identity/_static/image1.png)
+    ![Image of A S P dot Net configuration selection](migrating-an-existing-website-from-sql-membership-to-aspnet-identity/_static/image1.png)
```
Comment (Rick-Anderson) [inline aspnet/identity/overview/migrations/migrating-an-existing-website-from-sql-membership-to-aspnet-identity.md:30]: ASP

File: aspnet/identity/overview/migrations/migrating-an-existing-website-from-sql-membership-to-aspnet-identity.md [hunk 3]
```diff
@@ -24,33 +24,33 @@ For this tutorial, we will take a web application template (Web Forms) created u
...
 3. Open web.config and change the connection string from .SQLExpress to (LocalDb)v11.0. Remove 'User Instance=true' from the connection string.
 
-    ![](migrating-an-existing-website-from-sql-membership-to-aspnet-identity/_static/image3.jpg)
+    ![Image showing the connection string](migrating-an-existing-website-from-sql-membership-to-aspnet-identity/_static/image3.jpg)
 4. Open Server Explorer and verify that the table schema and data can be observed.
 5. The ASP.NET Identity system works with version 4.5 or higher of the framework. Retarget the application to 4.5 or higher.
 
-    ![](migrating-an-existing-website-from-sql-membership-to-aspnet-identity/_static/image5.png)
+    ![Image of A S P dot Net identity system and target framework](migrating-an-existing-website-from-sql-membership-to-aspnet-identity/_static/image5.png)
```
Comment (Rick-Anderson) [inline aspnet/identity/overview/migrations/migrating-an-existing-website-from-sql-membership-to-aspnet-identity.md:53]: ASP

**Review by Rick-Anderson — CHANGES_REQUESTED** [review]

File: aspnet/identity/overview/getting-started/introduction-to-aspnet-identity.md (hunk 1 above)
Comment (jmirabal80) [reply aspnet/identity/overview/getting-started/introduction-to-aspnet-identity.md:96]: I had thought screen readers interpreted ASP as an actual word and would try to pronounce it that way as opposed to the spacing variation of A S P in which each letter gets pronounced? Am I incorrect in that?

File: aspnet/identity/overview/migrations/migrating-an-existing-website-from-sql-membership-to-aspnet-identity.md (hunk 3 above)
Comment (Rick-Anderson) [reply aspnet/identity/overview/migrations/migrating-an-existing-website-from-sql-membership-to-aspnet-identity.md:53]: I think it's important to be consistent with the article where screen readers are also used.

File: aspnet/identity/overview/getting-started/introduction-to-aspnet-identity.md (hunk 1 above)
Comment (jmirabal80) [reply aspnet/identity/overview/getting-started/introduction-to-aspnet-identity.md]: change made

File: aspnet/identity/overview/migrations/migrating-an-existing-website-from-sql-membership-to-aspnet-identity.md (hunk 2 above)
Comment (jmirabal80) [reply aspnet/identity/overview/migrations/migrating-an-existing-website-from-sql-membership-to-aspnet-identity.md]: change made

File: aspnet/identity/overview/migrations/migrating-an-existing-website-from-sql-membership-to-aspnet-identity.md (hunk 3 above)
Comment (jmirabal80) [reply aspnet/identity/overview/migrations/migrating-an-existing-website-from-sql-membership-to-aspnet-identity.md]: change made

**Review by Rick-Anderson — APPROVED** [review]

**Pull request merged**